src/features/landing/components/AuthSection/AuthForm/AuthForm.tsx

```
@@ -1,177 +1,182 @@
-          </UiTypography>
-          <Stack sx={styles.inputsWrapper}>
-            <Stack sx={styles.inputWrapper}>
-              <UiTypography variant="medium14" sx={styles.inputTitle}>
-                {t('sign_up.form.name_input.label')}
+    return (
+      <Box
+        component="form"
+        data-testid="auth-form"
+        onSubmit={handleSubmit(onSubmit)}
+        ref={ref}
+        role="form"
```

Biome (1.9.4)

[error] The elements with the following roles can be changed to the following elements:
<form>


For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)